media-libs/libmpeg2/files/libmpeg2-0.5.1-armv4l.patch

```diff
@@ -1,6 +1,5 @@
-diff -urNp libmpeg2.orig/libmpeg2/motion_comp_arm_s.S libmpeg2/libmpeg2/motion_comp_arm_s.S
---- libmpeg2.orig/libmpeg2/motion_comp_arm_s.S	2008-07-09 21:16:05.000000000 +0200
-+++ libmpeg2/libmpeg2/motion_comp_arm_s.S	2009-11-20 19:55:22.000000000 +0100
+--- a/libmpeg2/motion_comp_arm_s.S	2008-07-09 21:16:05.000000000 +0200
++++ b/libmpeg2/motion_comp_arm_s.S	2009-11-20 19:55:22.000000000 +0100
```

Comment:
when I have to touch patches, I tend to remove the metadata likes dates too, because they dont add any value for us and make patches more noisy imo